src/components/appfooter.main.jsx

```diff
@@ -55,7 +55,7 @@ function AppFooterMain() {
         </div>
         <div className="footer-column social">
           <div className="title">
-            {intl.get('vestri')}
+            {intl.get('find-us-online')}
```

Comment:
Are we going to have the same title for both columns?

Reply:
No, it's three titles for three columns. There are two different "Find Us Online" columns but only one is visible at any time.